src/api/search/index.ts

```diff
@@ -21,7 +21,7 @@ export function convertToSearchNodePayload(
   criteria: SearchCriteria,
   organizationId?: number
 ): SearchNodePayload | undefined {
-  if (criteria === {}) {
+  if (Object.keys(criteria).length === 0) {
```

Comment:
changed because of 
```
src/api/search/index.ts:24:7 - error TS2839: This condition will always return 'false' since JavaScript compares objects by reference, not value.
```